### ISSUE TITLE
Gene.getInteractions() has apparently gone missing; throws exception when building webapp

### ISSUE THREAD

**Issue by sammyjava** [intermine/intermine#1479]
Just pulled the latest, greatest:
```
commit cbc82d403df2bcbc94253e92cc4b15f34c838f6c
Merge: f26c061 7ecbd2b
Author: julie-sullivan <julie@flymine.org>
Date:   Tue Dec 13 16:02:14 2016 +0000
```
And got this when building my webapp:
```
[javac] Compiling 98 source files to /home/intermine/intermine/bio/webapp/build/classes
[dependencies] warning: [options] bootstrap class path not set in conjunction with -source 1.6
[dependencies] /home/intermine/intermine/bio/webapp/src/org/intermine/bio/web/displayer/CytoscapeNetworkDisplayer.java:47: error: cannot find symbol
[dependencies]         	 size = ((Gene) object).getInteractions().size();
[dependencies]         	                       ^
[dependencies]   symbol:   method getInteractions()
[dependencies]   location: interface Gene
[dependencies] /home/intermine/intermine/bio/webapp/src/org/intermine/bio/web/displayer/CytoscapeNetworkDisplayer.java:51: error: cannot find symbol
[dependencies]         		 size = ((Protein) object).getGenes().iterator().next().getInteractions().size();
[dependencies]         		                                                       ^
[dependencies]   symbol:   method getInteractions()
[dependencies]   location: interface Gene
[dependencies] Note: Some input files use or override a deprecated API.
[dependencies] Note: Recompile with -Xlint:deprecation for details.
[dependencies] Note: Some input files use unchecked or unsafe operations.
[dependencies] Note: Recompile with -Xlint:unchecked for details.
[dependencies] 2 errors
[dependencies] 1 warning
```

**Comment by sammyjava** [comment]
BTW, this was easily fixed by deleting the offending source, ```intermine/bio/webapp/src/org/intermine/bio/web/displayer/CytoscapeNetworkDisplayer.java```. Not a displayer that I use.

**Comment by julie-sullivan** [comment]
Fixed by #1480

@sammyjava Let me know if you still have problems!

**Comment by sammyjava** [comment]
I have no more problems! NONE! :+1:
